docs/SUMMARY.md

```diff
@@ -21,6 +21,7 @@
 * [Python API](api-python.md)
 * [Jupyter API](api-jupyter.md)
 * [R API](api-r.md)
+* [Teams API](teams.md)
```

Comment:
this isn't an API as much as a new class of feature... IMHO create a new major top-level section called "Advanced Usage"